pkg/local-storage/member/controller/volume_migrate_task_worker.go

```diff
@@ -67,6 +65,30 @@ func (m *manager) processVolumeMigrate(vmName string) error {
 		return m.apiClient.Status().Update(context.TODO(), migrate)
 	}
 
+	lvmList := &apisv1alpha1.LocalVolumeMigrateList{}
+	if err := m.apiClient.List(context.Background(), lvmList); err != nil {
+		logCtx.WithError(err).Error("Failed to list VolumeMigrate from cache")
+		return err
+	}
+
+	if !migrate.Spec.Abort && (migrate.Status.State == "" ||
+		migrate.Status.State == apisv1alpha1.OperationStateSubmitted) {
+
+		var count int
+		for _, lvm := range lvmList.Items {
+			if lvm.Name != migrate.Name {
+				if lvm.Status.State == apisv1alpha1.OperationStateMigrateAddReplica ||
+					lvm.Status.State == apisv1alpha1.OperationStateMigrateSyncReplica ||
+					lvm.Status.State == apisv1alpha1.OperationStateMigratePruneReplica {
+					count++
+				}
+			}
+		}
+		if count >= m.migrateConcurrentNumber {
+			return fmt.Errorf("the number of concurrent migrations has been reached:%d", m.migrateConcurrentNumber)
+		}
+	}
+
```

Comment:
process for the limit tasks is ok, but this logic may remove to func `m.volumeMigrateSubmit`.

If the limit is reached, the state will not be flowed to the next stage `Submitted` and the migration will not start, you can put the speed limit information in the description field

Reply:
already edited